src/lib/ease/index.js

```diff
@@ -1,11 +1,36 @@
+// @flow
+
 // based on these https://gist.github.com/gre/1650294#gistcomment-1806616
 // all credit to @lindell
 
 import easeInOut from './lib/easeInOut';
 import easeOut from './lib/easeOut';
 import easeIn from './lib/easeIn';
 
-export default {
+export type EasingFunction = (time: number) => number;
+export type Easing = string | EasingFunction;
+
```

Comment:
Not sure if these should go here. Seemed like a natural thing to do but may be better in kind of central type files thing?